client/my-sites/domains/domain-management/transfer/transfer-to-other-site/index.jsx

```diff
@@ -51,9 +51,12 @@ class TransferToOtherSite extends React.Component {
 	}
 
 	isSiteEligible = site => {
+		// check if the iste is Atomic from the site options
+		const isAtomic = get( site, 'options.is_automated_transfer', false );
+
 		return (
 			site.capabilities.manage_options &&
-			! site.jetpack &&
+			! ( site.jetpack && ! isAtomic ) &&
```

Comment:
Would be a good idea to add a comment here. :) I work on these things, and it took me a while for this one. 😆 
```
// Non-atomic Jetpack sites
```